src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -7468,6 +7468,277 @@ public void SyntaxOffset_OutVarInMethod()
 ");
         }
 
+        [Fact]
+        public void SyntaxOffset_OutVarInInitializers_01()
+        {
+            var source =
+@"
+class C : A
+{ 
+    int x = G(out var x);
```

Comment:
>int x = G(out va [](start = 4, length = 16)

Nit: it's generally useful in these tests to write the source code on a single line (at least the part that contains the syntax of interest, such as the expression variables here) so that it's easier to check that the offsets are correct (one can see in the editor what column they start on).
It would indeed be even better if we had specialized test helper that used markers to indicate the syntax offsets.